forge-game/src/main/java/forge/game/GameAction.java

```diff
@@ -2773,19 +2771,18 @@ public void completeDungeon(Player player, Card dungeon) {
      *            the source
      * @return true, if successful
      */
-    public static boolean attachAuraOnIndirectEnterBattlefield(final Card source, Map<AbilityKey, Object> params) {
-        // When an Aura ETB without being cast you can choose a valid card to
-        // attach it to
-        final SpellAbility aura = source.getFirstAttachSpell();
+    private boolean attachAuraOnIndirectETB(final Card source, Map<AbilityKey, Object> params) {
+        // When an Aura ETB without being cast you can choose a valid card to attach it to
         if (!source.hasKeyword(Keyword.ENCHANT)) {
             return false;
         }
 
+        SpellAbility aura = source.getFirstAttachSpell();
         if (aura == null) {
-            return false;
+            // if it's not normally an aura
+            aura = new SpellAbility.EmptySa(ApiType.Attach, source);
```

Comment:
Add these SVars from source to Fake-Aura?
https://github.com/Card-Forge/forge/blob/b4af62eda71d1892ce4e13b3bf8adbda1f398f8c/forge-game/src/main/java/forge/game/card/CardFactory.java#L459-L467